go.sum

```diff
@@ -187,6 +191,12 @@ github.com/kr/pty v1.1.5/go.mod h1:9r2w37qlBe7rQ6e1fg1S/9xpWHSnaqNdHD3WcMdbPDA=
 github.com/kr/text v0.1.0/go.mod h1:4Jbv+DJW3UT/LiOwJeYQe1efqtUx/iVham/4vfdArNI=
 github.com/kr/text v0.2.0 h1:5Nx0Ya0ZqY2ygV366QzturHI13Jq95ApcVaJBhpS+AY=
 github.com/kr/text v0.2.0/go.mod h1:eLer722TekiGuMkidMxC/pM04lWEeraHUUmBw8l2grE=
+github.com/ldsec/medco-connector v1.0.0 h1:YTb4HubtYzARBYIEG2p+WInds8MFWrGOH55EWMCkkJ0=
+github.com/ldsec/medco-connector v1.0.0/go.mod h1:cNI6YOtSOCOKjiQrpN/lWBlpweAgWwwnqIksX0crV+k=
```

Comment:
Can you do a mod tidy here? And ensure the go.mod does not have external dependencies on any of `medco-connector`, `medco-loader` and `medco-unlynx`.

Reply:
should be ok now